README.md

```diff
@@ -67,6 +67,28 @@ affects more than the driver alone (for example, a feature which requires
 MongoDB server support), please submit your idea through the
 [MongoDB Feedback Forum](https://feedback.mongodb.com/forums/924286-drivers).
 
+## Bug Fix and Maintenance Policy
+
+Starting with Ruby Driver 2.16.0 we will no longer be backporting bug fixes
```

Comment:
This first sentence is not needed. Instead this section could start with new features and then move on to bug fixes, e.g.:

> New driver functionality is generally added in a backwards-compatible manner and results in new minor driver releases (2.x).

> Bug fixes are generally made on `master` first and are backported to the current minor driver release. Exceptions may be made on a case-by-case basis, for example security fixes may be backported to older stable branches.

> Only the most recent minor driver release is officially supported. Customers should use the most recent driver release in their applications.

Reply:
Done